controllers/install/lookout_controller.go

```diff
@@ -448,6 +562,135 @@ func createLookoutMigrationJob(lookout *installv1alpha1.Lookout) (*batchv1.Job,
 	return &job, nil
 }
 
+// createLookoutCronJob returns a batch CronJob or an error if the app config is not correct
+func createLookoutCronJob(lookout *installv1alpha1.Lookout) (*batchv1.CronJob, error) {
+	runAsUser := int64(1000)
+	runAsGroup := int64(2000)
+	terminationGracePeriodSeconds := int64(0)
+	if lookout.Spec.TerminationGracePeriodSeconds != nil {
+		terminationGracePeriodSeconds = int64(*lookout.Spec.TerminationGracePeriodSeconds)
+	}
+	allowPrivilegeEscalation := false
+	parallelism := int32(1)
+	completions := int32(1)
+	backoffLimit := int32(0)
+	env := lookout.Spec.Environment
+	volumes := createVolumes(lookout.Name, lookout.Spec.AdditionalVolumes)
+	volumeMounts := createVolumeMounts(GetConfigFilename(lookout.Name), lookout.Spec.AdditionalVolumeMounts)
+	dbPruningSchedule := "@hourly"
+	if lookout.Spec.DbPruningSchedule != nil {
+		dbPruningSchedule = *lookout.Spec.DbPruningSchedule
+	}
+
+	appConfig, err := builders.ConvertRawExtensionToYaml(lookout.Spec.ApplicationConfig)
```

Comment:
Codecov is reporting that you are missing some tests here.  I think it would be pretty easy to inject a failure here with a bad config.